plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -61,6 +64,76 @@ public void testDescribeTable()
         assertEquals(actualColumns, expectedColumns);
     }
 
+    @Test(dataProvider = "createTableColumnTypes")
+    public void testCreateTableWithEveryType(String createType, String expectedType)
+    {
+        String tableName = "test_types_table_" + randomTableSuffix();
```

Comment:
let's not use random suffix here. In other tests we are not using that and we have perparatory ` assertUpdate("DROP TABLE IF EXISTS " + tableName);` because of that.
If we use randomsuffixes we can pollute test environment and as far as I know we do not have any cleaner in place right now.